Training/Training/TestReversal.c

```diff
@@ -0,0 +1,132 @@
+// ------------------------------------------------------------------------------------------------
+// Assignment4 ~ A training program for new joiners at Metamation, Batch - Sept 2024.
+// Copyright (c) Gokul TS @Metamation India.
+// ------------------------------------------------------------------
+// TestReversal.c
+// C program to check test cases
+// ------------------------------------------------------------------------------------------------
+#include "Reversal.h"
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+/// <summary>Function to get string input from user</summary>

```

Comment:
It is known that this is a function.
```suggestion 
/// <summary>Gets string input from user</summary>

```